tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -169,6 +169,8 @@ public function testReturnQuotedAssets(): void
             [
                 'CREATE TABLE dbal91_something (id INT NOT NULL, "table" INT DEFAULT NULL, PRIMARY KEY(id))',
                 'CREATE INDEX IDX_A9401304ECA7352B ON dbal91_something ("table")',
+                'ALTER TABLE dbal91_something ADD CONSTRAINT something_input FOREIGN KEY ("table")'
+                    . ' REFERENCES dbal91_something (id) ON UPDATE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE',
```

Comment:
This change should have been made in #5416 if we wanted to be absolutely deprecation-free but we don't.